### ISSUE TITLE
Add membersLimit filter in ChannelListQuery

### ISSUE THREAD

**Issue by tokirin** [GetStream/stream-chat-swift#1688]
## What are you trying to achieve?
Hello.
It has been observed that the performance of synchronizing `ChatChannelListController` becomes significantly slower when there are many channels with a large number of user. 
So, I want to optimize performance by adding a `membersLimit` filter to the `ChannelListQuery`
This is because it is not necessary to bring individual member information to the view of channel list.

## If possible, how can you achieve this currently?

Our team directly forked this repo and modified `ChannelListQuery` to fix it

## What would be the better way?
[Here is the commit](https://github.com/educast/stream-chat-swift/commit/f208336ce5f296d80b7ab33692fd6ac6a6b99209)
### Usage
```Swift
      myChannelsQuery = ChannelListQuery(
        filter: .containMembers(userIds: [userId]),
        pageSize: 15,
        messagesLimit: 25,
        membersLimit: 1
      )
```

After applying that commit, I saw a huge performance improvement. 
Would you apply that for next update? 

## GetStream Environment
**GetStream Chat version: 4.4.0
**GetStream Chat frameworks:** StreamChat
**iOS version: 15.x, 14.x
**Swift version: 5.1
**Xcode version: 13.1
**Device: all of iPhone, iPad supporting iOS 14+

**Comment by b-onc** [comment]
Hello @tokirin 

That's a good addition, thank you for suggesting it. Since we're in the process of rolling out the next version, this can't go into it, but we'll add it into the version after that one.

**Comment by tbarbugli** [comment]
@tokirin this feature was added and released on 4.7.0
